iostat/iostat.go

```diff
@@ -65,19 +66,25 @@ func New() (*IOSTAT, error) {
 }
 
 // CollectMetrics returns metrics from iostat
-func (iostat *IOSTAT) CollectMetrics(mts []plugin.PluginMetricType) ([]plugin.PluginMetricType, error) {
+func (iostat *IOSTAT) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, error) {
 	_, data, err := iostat.run(mts)
 	if err != nil {
 		return nil, err
 	}
-	metrics := make([]plugin.PluginMetricType, len(mts))
+	metrics := make([]plugin.MetricType, len(mts))
 	hostname, _ := os.Hostname()
 	for i, m := range mts {
-		if v, ok := data[joinNamespace(m.Namespace())]; ok {
-			metrics[i] = plugin.PluginMetricType{
+		tags := m.Tags()
```

Comment:
I agree, also I think I need to provide description and units whenever possible. Applies to all opened PRs ...